FROST-Server.Auth.Basic/src/main/java/de/fraunhofer/iosb/ilt/frostserver/auth/basic/BasicAuthProvider.java

```diff
@@ -38,6 +38,9 @@ public class BasicAuthProvider implements AuthProvider, LiquibaseUser, ConfigDef
     @DefaultValueBoolean(false)
     public static final String TAG_AUTO_UPDATE_DATABASE = "autoUpdateDatabase";
 
+    @DefaultValueBoolean(true)
+    public static final String TAG_PLAIN_TEST_PASSWORD = "plainTextPassword";
```

Comment:
Typo in the name of the constant? (TEST -> TEXT)

Reply:
I fixed the typo.